src/components/Nav.tsx

```diff
@@ -0,0 +1,111 @@
+import styled from 'styled-components';
+import css from '@styled-system/css';
+import themeGet from '@styled-system/theme-get';
+import {
+  margin,
+  MarginProps,
+  padding,
+  PaddingProps,
+  height,
+  HeightProps,
+  minHeight,
+  MinHeightProps,
+  width,
+  WidthProps,
+  minWidth,
+  MinWidthProps,
+  variant,
+  compose
+} from '@modulz/radix-system';
+
+type NavProps = MarginProps & PaddingProps & HeightProps & MinHeightProps;
+
+const styleProps = compose(margin, padding, height, minHeight);
+
+export const Nav = styled('nav')<NavProps>(
+  css({
```

Comment:
Static style should be in sass file and shouldn't be written as CSS in JS IMO 🤔

Reply:
CSS-in-JS is a wide debate and I don't think that it should be handle in this particular PR since the whole project is handle this way. Feel free to open an issue.